CONTRIBUTING.md

```diff
@@ -115,3 +115,9 @@ docs: fix typo in readme file
 fix(core): add check for atoms with equal ids
 docs(react): update redux migration guide
 ```
+
+## Tests rules
```

Comment:
You could remove it now, `npm run test` should work well now and it obvious. Btw, remove the readme changes pls